examples/pa_engine_multiple_unit_linked_templated_component_example.py

```diff
@@ -0,0 +1,251 @@
+    config.retries = Retry(total=3, status=3, status_forcelist=frozenset([429, 503]), backoff_factor=2,
+                           raise_on_status=False)
+
+    api_client = ApiClient(config)
+
+    try:
+        column_name = "Port. Average Weight"
+        column_category = "Portfolio/Position Data"
+        column_statistic_name = "Active Weights"
+
+        group_category = "JP Morgan CEMBI "
+        group_name = "Country - JP Morgan CEMBI "
+
+        component_document = "PA_DOCUMENTS:DEFAULT"
+        component_name = "Weights"
+        component_category = "Weights / Exposures"
+
+        linked_pa_template_directory = "Personal:LinkedPATemplates/"
+        templated_pa_component_directory = "Personal:TemplatedPAComponents/"
+
+        portfolio = "BENCH:SP50"
+        benchmark = "BENCH:R.1000"
+        startdate = "20180101"
+        enddate = "20181231"
+        frequency = "Monthly"
+        currencyisocode = "USD"
+        componentdetail = "GROUPS"
+        directory = "Factset"
+
+        # uncomment the below code line to setup cache control; max-stale=0 will be a fresh adhoc run and the max-stale value is in seconds.
+        # Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older. 
+        # cache_control = "max-stale=0"
+        
+        # get column id
+        columns_api = ColumnsApi(api_client)
+        column = columns_api.get_pa_columns(
+            name = column_name,
+            category = column_category,
+            directory = directory
+        )
+        column_id = list(column[0].data.keys())[0]
+
+        # get column statistics id
+        column_statistics_api = ColumnStatisticsApi(api_client)
+        get_all_column_statistics = column_statistics_api.get_pa_column_statistics()
+        desired_column_statistic = ColumnStatistic(name=column_statistic_name)
+        column_statistic_id = [id for id in list(
+            get_all_column_statistics[0].data.keys()) if get_all_column_statistics[0].data[id] == desired_column_statistic][0]
+
+        # create columns parameter
+        columns = [PACalculationColumn(id=column_id, statistics=[column_statistic_id])]
+
+        # get group id
+        groups_api = GroupsApi(api_client)
+        groups = groups_api.get_pa_groups()
+        desired_group = Group(category=group_category, directory=directory, name=group_name)
+        group_id = [id for id in list(
+            groups[0].data.keys()) if groups[0].data[id] == desired_group][0]
+
+        # create groups parameter
+        groups = [PACalculationGroup(id=group_id)]
+
+        # get parent component id
+        components_api = ComponentsApi(api_client)
+        components = components_api.get_pa_components(document=component_document)
+        desired_component = ComponentSummary(
+            name=component_name, category=component_category
+        )
+        parent_component_id = [id for id in list(
```

Comment:
This comparison will fail if we add more fields in components that we receive in the API response. Can you compare using `name` and `category` instead of creating `ComponentSummary`?

Reply:
fixed https://github.com/factset/analyticsapi-engines-python-sdk/pull/62/commits/ac8e357b3e85a8dd702a11ab72db5dbcc1acf804